hilo.rb

```diff
@@ -1,20 +1,6 @@
-random_number = rand(1..100)
-puts "Enter a number between 1 and 100, then press enter to submit your guess. You have 6 chances to guess the correct number"
-input = gets.chomp
-guess = input.to_i
-count = 0
-while count < 5
-    count = count + 1
-    if guess.to_s != input
-        puts "Your guess is not a whole number. Try again"
-    elsif guess > random_number
-        puts "Your guess is too high. Try again"
-    elsif guess < random_number
-        puts "Your guess is too low. Try again"
-    else
-        break
-    end
-    input = gets.chomp
-    guess = input.to_i
-end
-puts (guess == random_number) ? "Correct! You win" : "Sorry, you ran out of guesses. The number was #{random_number}"
+require_relative 'new_game'
+
+new_game = NewGame.new
```

Comment:
Just curious why you opted to have the `NewGame` class in a different file.  On the side there is also an option to have multiple classes in one file: https://docs.ruby-lang.org/en/3.0/syntax/modules_and_classes_rdoc.html.  

Also, why not use `private methods` instead?  Just something to think about.

Reply:
I started with it in the same file but thought it was tidier to separate it out from thee running file. Happy to change if it makes more sense to keep them in the same file?

Reply:
Yep makes sense to use private methods - will update

Reply:
Would you suggest using a class or a module for this game? I don't fully understand the difference after reading that documentation